src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -14,6 +14,27 @@ import { COLUMN_KEYS } from './constants';
 import { ROUTES } from '../../navigation/constants';
 import { SETTINGS_KEYS } from '../../settings';
 
+export const vaccinesAdminInitialiser = () => {
+  const backingData = UIDatabase.objects('Fridge');
+  const data = backingData.slice();
+  const sortedData = sortDataBy(data, COLUMN_KEYS.CODE, false);
+
+  return {
+    backingData,
+    data: sortedData,
+    dataState: new Map(),
+    sortKey: COLUMN_KEYS.CODE,
+    isAscending: true,
+    keyExtractor: recordKeyExtractor,
+    searchTerm: '',
+    modalKey: '',
+    columns: getColumns('fridges'),
+    route: ROUTES.VACCINES_ADMIN,
+    dataSet: 'fridges',
+    filterDataKeys: ['description', 'code'],
+  };
+};
```

Comment:
wanted to use selectors and things to get data but just went with this as it's consistent/easier 🤷